admin/create-theme/theme-zip.php

```diff
@@ -172,7 +172,7 @@ public static function add_templates_to_zip( $zip, $export_type, $new_slug ) {
 
 			// Write the template content
 			$zip->addFromStringToTheme(
-				$template_folders['wp_template'] . DIRECTORY_SEPARATOR . $template->slug . '.html',
+				$template_folders['wp_template'] . '/' . $template->slug . '.html',
```

Comment:
We could also use [`path_join`](https://developer.wordpress.org/reference/functions/path_join/). I find that it's easier to figure out what's going at a glance, and also provides protection against extra slashes that might show up.

```suggestion
				path_join( $template_folders['wp_template'], $template->slug . '.html' ),
```